variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "bucket_tags" {
```

Comment:
Sorry that I did not catch this before, but do you mind moving this variable definition down to be alongside the other bucket-related variables? For example below the [`bucket_sse_key_arn` variable](https://github.com/lacework/terraform-aws-agentless-scanning/pull/105/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eL123)?

Reply:
No worries, good idea.